### PR TITLE
query hlr api

### PR DESCRIPTION
**Pull request by babasahebk**
## Ticket
https://jira.bqsoft.com/browse/ATOM-8508

## Description
ITG Query HLR API

### CODE REVIEW

File: lib/conduit/plintron/actions/query_hlr.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  class QueryHlr < Conduit::Driver::Plintron::Base
+    optional_attributes :imsi, :iccid, :mdn
+    operation :QUERY_HLR
+
+    def initialize(options = {})
+      super
+
+      unless device_requirements_met?
+        raise ArgumentError,
+          "You must provide one of the following attributes: imsi, iccid, mdn"
```
Comment (dougperez) [inline lib/conduit/plintron/actions/query_hlr.rb]: I think we have something like this already for optional required attributes

Comment (pedromartinez) [reply lib/conduit/plintron/actions/query_hlr.rb]: Oh yeah, an array, right?

`required_attributes [:imsi, :iccid, :mdn]` maybe?

Comment (dougperez) [reply lib/conduit/plintron/actions/query_hlr.rb]: Yeah, example:
https://github.com/conduit/conduit-cogotech/blob/f82093280aec56988fa9f8d22b8e9316116b5aa0/lib/conduit/cogotech/actions/restore.rb#L5

Comment (babasahebk) [reply lib/conduit/plintron/actions/query_hlr.rb]: updated it

File: lib/conduit/plintron/decorators/activate.rb [hunk 2]
```diff
@@ -2,5 +2,8 @@
 
 module Conduit::Plintron::Decorators
   class ActivateDecorator < Base
+    def asynchronous
```
Comment (dougperez) [inline lib/conduit/plintron/decorators/activate.rb]: Are we using this flag somehwere?

Usually we use `true/false` instead of `1/nil`

**Pull request merged**